src/sentry/tasks/post_process.py

```diff
@@ -1056,6 +1056,28 @@ def fire_error_processed(job: PostProcessJob):
         )
 
 
+def sdk_crash_monitoring(job: PostProcessJob):
+    from sentry.utils.sdk_crashes.sdk_crash_detection import sdk_crash_detection
+
+    if job["is_reprocessed"]:
+        return
+
+    event = job["event"]
+
+    if not features.has("organizations:sdk-crash-detection", event.project.organization):
+        return
+
+    if settings.SDK_CRASH_DETECTION_PROJECT_ID is None:
+        logger.warning(
+            "SDK crash detection is enabled but SDK_CRASH_DETECTION_PROJECT_ID is not set."
+        )
+        return None
+
+    with metrics.timer("post_process.sdk_crash_monitoring.duration"):
+        with sentry_sdk.start_span(op="tasks.post_process_group.sdk_crash_monitoring"):
```

Comment:
I believe you can start these two context managers at the same time:
```suggestion
    with metrics.timer("post_process.sdk_crash_monitoring.duration"), sentry_sdk.start_span(op="tasks.post_process_group.sdk_crash_monitoring"):
```

Reply:
Thanks 😃